acme/tests/client_test.py

```diff
@@ -1157,7 +1157,7 @@ def test_urllib_error(self):
 
         # Requests Library Exceptions
         except requests.exceptions.ConnectionError as z: #pragma: no cover
-            self.assertTrue("'Connection aborted.'" in str(z) or "[WinError 10061]" in str(z))
+            self.assertIn("'Connection aborted.'" in str(z) or "[WinError 10061]", str(z))
```

Comment:
The current test failure is due to this change.

The `assertTrue` was `A in str or B in str`, so I don't think we can use a single `assertIn` here.

Reply:
Thanks! I used a regex to replace them, and it was obviously a bit too aggressive :/